レコード追加API作成

Coms7 opened the pull request:
値を渡すまではできたものの、リダイレクトがわからず。

Coms7 committed:
レコード追加API作成

値を渡すまではできたものの、リダイレクトがわからず。

bamv-ikeda commented on ZipCodeApi.rb:
```diff
@@ -13,15 +13,14 @@
- post '/insert' do
-    zip_codes = params[:zip_codes]
-    statement = client.prepare('INSERT INTO zip_codes(zip_code, prefecture, city, town_area, created_at, updated_at)
+post '/insert' do
```
insertでも問題があるわけではないですが、勉強の意味も兼ねてrest apiにしてみましょう
`/insert` -> `/zip_codes`
https://qiita.com/masato44gm/items/dffb8281536ad321fb08

Coms7 replied:
わかりました！やってみます！

Coms7 committed:
create search&derete

updateは途中です

Coms7 committed:
create update

まだnot foundになります。 動作は確認できていません。

Coms7 committed:
それぞれのメソッドをRESTに変更

Coms7 committed:
URI変更、戻り値をjson形式に変更

URLとメソッドの組み合わせでRESTなルート設計に変更。
戻り値をjsonにして返すように変更。
hashに詰める過程はひとまず簡単な方法にし、動作確認を優先しました。
